lib/models/package-info-cache/index.js

```diff
@@ -445,7 +446,7 @@ class PackageInfoCache {
 
       let nodeModulesPath = endsWithNodeModules
         ? currPath
-        : `${currPath}${path.sep}node_modules`;
+        : `${currPath}/node_modules`;
```

Comment:
Regardless of the other fixes, I *really* don't like this usage of / in the path. I think it was @twokul that suggested using path.join(currPath, 'node_modules) here instead. I think that's much more preferable. It doesn't mean the other changes to always convert to posix paths are not good, but explicitly relying on them by inserting / as a constant just seems wrong.

Reply:
In internal code, I prefer to leave input as-is, until the normalization is actually required. This avoids wack-a-mole, as only code which cares always normalizes before utilization.